Gruntfile.js

```diff
@@ -390,6 +390,12 @@ module.exports = (grunt) => {
     fs.readdirAsync('./nmclicktests')
       .then((files) => files.filter((file) => file.startsWith("spec.")))
       .then((tests) => {
+        const genericIndx = tests.indexOf("spec.generic.js")
```

Comment:
Is the order important?

Reply:
For the parallel tests yes.  `spec.generic.js` is the longest test we have.  by having it start first we are ensuring that best case scenario runtimes is runtime of `generic` not `generic + another test(s)`.